src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceAboutToSubmitHandlerTest.java

```diff
@@ -195,6 +198,58 @@ public void givenAValidPostponementRequest_thenSscsDocumentTypeIsPostponementReq
                 .anyMatch(note -> note.getValue().getNoteDetail().equals("Request Detail Test")), is(true));
     }
 
+    private void setUpCaseDataForSetAside() {
+        sscsCaseData.setState(State.DORMANT_APPEAL_STATE);
+        sscsCaseData.getFurtherEvidenceAction().setValue(
+                new DynamicListItem(FurtherEvidenceActionDynamicListItems.SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getCode(),
+                        FurtherEvidenceActionDynamicListItems.SEND_TO_INTERLOC_REVIEW_BY_JUDGE.getLabel())
+        );
+        ScannedDocument scannedDocument = ScannedDocument.builder()
+                .value(ScannedDocumentDetails.builder().type(SET_ASIDE_APPLICATION.getValue())
```

Comment:
Would be preferable to make this extensible for the other Post hearing types. I have taken the approach of set up everything except set aside-specific in the before each, then plug in the type in a parameterized test. This way future us can easily add Correction etc.

Reply:
Agreed. Will do, checked upcoming post hearing tickets and the outcomes are pretty much the same sans the document type, so plumbing this in now will save even more time down the line.

Reply:
perfect